src/main/java/com/redislabs/WriteBehind.java

```diff
@@ -45,7 +47,7 @@ private static void registerOnChanges(String mapping) {
       String key = record.getKey();
       String[] keySplit = key.split(":");
       
-      Stream<String> commandStream = Stream.of("XADD", "stream", "*", "entityName", keySplit[0],  "id", keySplit[1]);
+      Stream<String> commandStream = Stream.of("XADD", String.format("_Stream-{%s}", GearsBuilder.hashtag()), "*", "entityName", keySplit[0],  "id", keySplit[1]);
```

Comment:
@MeirShpilraien I would have try to avoid creating this string on each call

Reply:
Done